src/backend/marsha/core/tests/api/playlists/test_update.py

```diff
@@ -13,6 +13,8 @@
 class PlaylistUpdateAPITest(TestCase):
     """Test the update API for playlist objects."""
 
+    maxDiff = None
```

Comment:
![internet-wow](https://user-images.githubusercontent.com/10722639/199452858-82bdd3ba-b7f5-4834-ae10-72f1fdbffa08.gif)

Reply:
And so what ? Should I make a change ?

Reply:
I have no idea, I don't know this does

Reply:
It allow us to show the full diff when an assert is false: https://docs.python.org/3/library/unittest.html#unittest.TestCase.maxDiff